MVVMRxSwiftDemoTests/Sources/Specs/Presentation/Modules/Login/LoginViewModelSpec.swift

```diff
@@ -87,7 +87,18 @@ final class LoginViewModelSpec: QuickSpec {
 
                     expect(viewModel.output.didLogin.compactMap { $0 as? HomeViewModelProtocolMock })
                         .events(scheduler: scheduler, disposeBag: disposeBag)
-                        .to(equal([.completed(100)]))
+                        .to(equal([]))
+                }
+                
+                it("returns output with correct errorMessage") {
+                    scheduler.scheduleAt(50) {
+                        viewModel.input.didTapLoginButton(email: "dummyaccount@nimblehq.co", password: "12345678")
+                    }
+                    viewModel.output.didLogin.asObservable().bind { _ in }.disposed(by: disposeBag)
```

Comment:
Oh, so we need to use `viewModel.output.didLogin` first, to make to RxErrorTracker works. Good catch 🎉

Reply:
And I prefer this for shorter
```suggestion
                    viewModel.output.didLogin.drive().disposed(by: disposeBag)
```

Reply:
> Oh, so we need to use `viewModel.output.didLogin` first, to make to RxErrorTracker works. Good catch 🎉

@blyscuit helped me 😄

Reply:
@blyscuit @llleyelll please bring these topics into the ios-chapter channel next time, so people can help or learn from you guys 😁